src/calendar/__tests__/calendar.test.tsx

```diff
@@ -34,7 +34,7 @@ function renderCalendar(props: CalendarProps = defaultProps) {
 
 function findCalendarWeekdays(wrapper: CalendarWrapper) {
   return wrapper
-    .findAll(`.${styles['calendar-date-header']} :not(.${screenreaderOnlyStyles.root})`)
+    .findAll(`.${styles['calendar-date-header']} [aria-hidden="true"]`)
```

Comment:
The old selector fails with `unknown pseudo-class selector` after migration to the newer Jest.

Reply:
`:not` is a standard selector, supported in browsers and used in our test-utils. It should work in our tests, otherwise we won't be able to use our own test-utils.

Need a proper dive to understand why it stopped working and fix it without changing the code

Reply:
Updated. There were syntax errors in the prev selector tolerated by the older version of jest.

Reply:
That's the intended fix 👍